flatfs_test.go

```diff
@@ -400,6 +401,50 @@ func testBatchDelete(dirFunc mkShardFunc, t *testing.T) {
 
 func TestBatchDelete(t *testing.T) { tryAllShardFuncs(t, testBatchDelete) }
 
+func TestNoCluster(t *testing.T) {
+	tempdir, cleanup := tempdir(t)
+	defer cleanup()
+
+	fs, err := flatfs.New(tempdir, flatfs.NextToLast(1), false)
+	if err != nil {
+		t.Fatalf("New fail: %v\n", err)
+	}
+
+	r := rand.New()
+	N := 1024 // should be divisible by 32 so the math works out
```

Comment:
I prefer a number a direct number.  If we use 32* I will have to stop and do the math to figure the numbers of key used.  The comment also makes it explicit.  With just a 32*X then I would wonder why it is written like that.  @whyrusleeping what's your view.